### ISSUE TITLE
Combine issues/PRs comments into one comment

### ISSUE THREAD

**Issue by bubkoo** [dhoulb/multi-semantic-release#41]
Each release add a comment to the related issue and pr.  Combine the comments into one making a better life.

![image](https://user-images.githubusercontent.com/6045824/97839476-3d9eb380-1d1d-11eb-9b13-f4c262e871f3.png)

**Comment by antongolub** [comment]
Yep, this is really annoying. As a temporary workaround we just recommend to disable github plugin comments.
```js
module.exports = {
  branch: 'master',
  plugins: [
    '@semantic-release/commit-analyzer',
    '@semantic-release/release-notes-generator',
    '@semantic-release/changelog',
    '@semantic-release/npm',
    [
      '@semantic-release/github',
      {
        successComment: false,
        failComment: false
      }
    ],
    '@semantic-release/git'
  ]
}
```

**Comment by esatterwhite** [comment]
This is probably related to the same underlying issue. I notice that when a new packages is added to an existing monorepo the initial release of the package results addeding the release success comment on every or before it. 

Like the commits aren't filtered. It updates months worth of prs and GitHub sends out dozens of email notifications

**Comment by esatterwhite** [comment]
Is this what the `--first parent` option aims to solve?

**Comment by antongolub** [comment]
@esatterwhite, 
 
`--first-parent` is used to filter commits by release branch only (master, main).

It seems something went wrong here.
https://github.com/semantic-release/github/blob/d70e766d275b856dbe95f48358dcb9e6025748a7/lib/success.js#L43
```js
  } else {
    const parser = issueParser('github', githubUrl ? {hosts: [githubUrl]} : {});
    const releaseInfos = releases.filter((release) => Boolean(release.name));
    const shas = commits.map(({hash}) => hash);

    const searchQueries = getSearchQueries(`repo:${owner}/${repo}+type:pr+is:merged`, shas).map(
      async (q) => (await github.search.issuesAndPullRequests({q})).data.items
    );
```
We need to debug this point to make sure, that only filtered commits are passed to plugin method.

**Comment by esatterwhite** [comment]
@antongolub Making a new issue so this doesn't get lost

**Comment by antongolub** [comment]
![image](https://user-images.githubusercontent.com/5288046/144063458-3a97f8f1-6413-4e2a-9a03-e1f541ae5389.png)
